### PR TITLE
Fixed buttons color at smart contract section

### PR DESCRIPTION
**Pull request by YegorSan**
This PR is related to #2652 issue.

@vbaranov As you requested, I've re-created this PR to master branch, and now I am closing #2655 PR with the same update which was added to front-update branch. 

## Motivation

* Different color of buttons at smart contract page. 

### Bug Fixes

* Fixed color of a "Query" buttons at Read Contract section.

## Checklist for your PR

  - [ ] I added an entry to `CHANGELOG.md` with this PR
  - [ ] If I added new functionality, I added tests covering it.
  - [ ] If I fixed a bug, I added a regression test to prevent the bug from silently reappearing again.
  - [ ] I checked whether I should update the docs and did so if necessary
  - [ ] If I added/changed/removed ENV var, I should update the list of env vars in https://github.com/poanetwork/blockscout/blob/master/docs/env-variables.md to reflect changes in the table here https://poanetwork.github.io/blockscout/#/env-variables?id=blockscout-env-variables. I've set `master` in the `Version` column.
  - [ ] If I add new indices into DB, I checked, that they don't redundant with PGHero or other tools

### CODE REVIEW

**Comment by vbaranov** [comment]
@YegorSan please add CHANGELOG entry

**Comment by YegorSan** [comment]
@vbaranov Hello. Added new CHANGELOG entry.

**Comment by YegorSan** [comment]
@vbaranov Hello Victor. Merging conflict was resolved. Please review.

**Comment by vbaranov** [comment]
@YegorSan 
>Merging conflict was resolved. Could you please review?

Yes, but CHANGELOG entry is missing

**Comment by YegorSan** [comment]
@vbaranov Changelog is updated.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ## Current
 
 ### Features
+- [#2671](https://github.com/poanetwork/blockscout/pull/2671) - fixed buttons color at smart contract section
```
Comment (vbaranov) [inline CHANGELOG.md]: @YegorSan it is not a feature. Let's move to *Fix* section

Comment (YegorSan) [reply CHANGELOG.md]: @vbaranov Moved new entry to fixes.

**Pull request merged**